pkg/config/config.go

```diff
@@ -825,6 +825,16 @@ func (c *Config) ToFile(path string) error {
 	return os.WriteFile(path, b, 0o644)
 }
 
+// LogConfig pretty prints current config
+func (c *Config) LogConfig() error {
```

Comment:
Turn this into a helper that prints the current configuration, formatted as TOML, as a string value, for example:

```go
// ToString encodes the config into a string value.
func (c *Config) ToString() (string, error) {
        configBytes, err := c.ToBytes()
        if err != nil {
                return "", err
        }

        return string(configBytes), nil
}
```

This fits right next to the `ToBytes()`. Then, you can use it in the main function to print the configuration, without the need to have logging added to such a helper, which it shouldn't really. So, this would then be, for example:

```go
// Print the current configuration.
tomlConfig, err := config.ToString()
if err != nil {
	logrus.Errorf("Unable to print current configuration: %v", err)
}
logrus.Infof("Current CRI-O configuration:\n%s", tomlConfig)
```

Or with an if-else to avoid printing the extra log line with an empty value in case of an error:

```go
// Print the current configuration.
tomlConfig, err := config.ToString()
if err != nil {
	logrus.Errorf("Unable to print current configuration: %v", err)
} else {
	logrus.Infof("Current CRI-O configuration:\n%s", tomlConfig)
}
```